Disable vulkan support.
Handbrake doesn't require Vulkan but if it is installed in build environment
(for other ports like QT5),  It fails to build Handbrake, because following
definitions has been removed since vulkan-headers-1.3.238.

* VK_EXT_VIDEO_DECODE_H264_EXTENSION_NAME
* VK_EXT_VIDEO_DECODE_H265_EXTENSION_NAME

--- contrib/ffmpeg/module.defs.orig	2022-12-28 08:43:46 UTC
+++ contrib/ffmpeg/module.defs
@@ -37,6 +37,7 @@ FFMPEG.CONFIGURE.extra = \
     --enable-libzimg \
     --disable-hwaccels \
     --disable-vdpau \
+    --disable-vulkan \
     --disable-encoders \
     --enable-libmp3lame \
     --enable-encoder=aac \
